NetKAN/Arkas.netkan

```diff
@@ -0,0 +1,24 @@
+{
+	"identifier"	:	"Arkas",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"spec_version"	:	"v1.4",
+	"license"  	:	"CC-BY-NC-SA-4.0",
+	"comment"	:	"PlanetPack",
+	"resources" :
```

Comment:
hmmm.  Sorry, I was rushing to a dr. appt.

Ok, changes made

Thanks

and, in general, a big thanks for all the work you do for this project.


JBB

On 10/30/2015 7:38 AM, Olympic1 wrote:
>
> In NetKAN/Arkas.netkan 
> <https://github.com/KSP-CKAN/NetKAN/pull/2505#discussion_r43493212>:
>
> > @@ -0,0 +1,24 @@
> > +{
> > +	"identifier"	:	"Arkas",
> > +	"$kref"		:	"#/ckan/kerbalstuff/1158",
> > +	"spec_version"	:	"v1.4",
> > +	"license"  	:	"CC-BY-NC-SA-4.0",
> > +	"comment"	:	"PlanetPack",
> > +	"resources" :
>
> Also forgot to remove the |resources| section
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/KSP-CKAN/NetKAN/pull/2505/files#r43493212>.
>